src/auth/src/credentials/user_credential.rs

```diff
@@ -27,11 +27,14 @@ const OAUTH2_ENDPOINT: &str = "https://oauth2.googleapis.com/token";
 pub(crate) fn creds_from(js: serde_json::Value) -> Result<Credential> {
     let au =
         serde_json::from_value::<AuthorizedUser>(js).map_err(CredentialError::non_retryable)?;
+
+    let endpoint = au.token_uri.unwrap_or_else(|| OAUTH2_ENDPOINT.to_string());
+
```

Comment:
I do not see any new tests exercising the new functionality. Did I miss them?

Reply:
Added